Tests/UI/CardViewControllerTests.swift

```diff
@@ -322,26 +322,6 @@ class CardViewControllerTests: XCTestCase {
         XCTAssertEqual(nFadedCard, 0)
     }
 
-    func test_onTapDoneCardButton_checkoutAPIClientReturnsError_delegateCalledWithFailure() {
-
```

Comment:
No need to support Card view controller, it will be deleted soon

Reply:
Thank you, was hoping you won't ask me to fix this test as I actually tried and would need to understand that VC (which I know we'll delete) 😄 🙏